payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/LpmRepository.kt

```diff
@@ -0,0 +1,275 @@
+package com.stripe.android.ui.core.elements
+
+import android.os.Parcelable
```

Comment:
This is code that was just moved from the PaymentSheet module's SupportedPaymentMethod.  It contains just the data that defines an LPM.